src/activity/views/configurationView/ConfigureActivityView.tsx

```diff
@@ -0,0 +1,492 @@
+"use client";
+
+import type React from "react";
+import { useState } from "react";
+import { motion } from "framer-motion";
+import {
+  FaCog,
+  FaCalendarAlt,
+  FaClock,
+  FaGraduationCap,
+  FaChartLine,
+  FaSave,
+  FaEye,
+  FaBook,
+  FaInfoCircle,
+  FaExclamationTriangle,
+} from "react-icons/fa";
+import type {
+  ConfigurationActivity,
+  ConfigurationErrors,
+} from "../../types/Configuration.type";
+import Button from "../../../shared/components/Button/ButtonComponent";
+import Input from "../../../shared/components/Input/InputComponent";
+import Card from "../../../shared/components/Card/CardComponent";
+import Badge from "../../../shared/components/Badge/BadgeComponent";
+import styles from "./ConfigureActivityView.module.css";
+import { useNavigate, useParams } from "react-router-dom";
+import { useSubject } from "../../../admin/hooks/useSubject";
+import { useConfigurationActivity } from "../../hooks/useConfigurationActivity";
+
+const ConfigureActivityView: React.FC = () => {
+  const [configuration, setConfiguration] = useState<ConfigurationActivity>({
+    description: "",
+    startDate: "",
+    endDate: "",
+    dificulty: "",
+    maxTime: 30,
+    subjectId: 0,
+  });
```

Comment:
Aca podríamos hacer una función que setee todo default (sería tipo el resetForm() de la parte de admin)

```suggestion
  const getDefaultConfiguration = (): ConfigurationActivity => {
  const now = new Date();
  const inOneHour = new Date(now.getTime() + 60 * 60 * 1000);

  return {
    description: "",
    startDate: now.toISOString().slice(0, 16),
    endDate: "",
    dificulty: "",
    maxTime: 30,
    subjectId: 0,
  };
};

const [configuration, setConfiguration] = useState<ConfigurationActivity>(getDefaultConfiguration());
```